mypy.ini

```diff
@@ -9,6 +9,7 @@ check_untyped_defs = True
 warn_unreachable = True
 strict_equality = True
 ignore_missing_imports = True
+exclude = posthog/hogql/grammar/
```

Comment:
Keeping the generated grammar files as they are, and ignoring all errors, autoformatting, etc on them.